README.md

```diff
@@ -73,16 +160,53 @@ If you have a Snyk token, add that as an environment variable:
      For example, a value of `$JENKINS_HOME/tools/hudson.tasks.Maven_MavenInstallation/<Maven Name from Step 1>/bin/mvn`.
   4. Include **Invoke top-level maven targets** as a build step by specifying the Maven version, and add **clean install** as a goal for the new pipeline item.
   
-#### Option II- As a pipeline task
+#### Option 2 - As a pipeline task
 - Its just a single line that you need to add in your pipeline script.
 `rhdaAnalysis file:'manifest file path', consentTelemetry:true`
 The value description remains the same as provided in the Option I.
 User can also use the pipeline snippet generator to generate the command.
   ![](./images/pipeline.png)
+
+
+##### Example basic pipeline
+**Note: This basic declarative pipeline assumes that package managers binaries preinstalled in the pipeline' invoking machine (jenkins master or jenkins agent) **
+```java
+
+pipeline {
+    agent any
+    stages {
+        stage('Checkout') {
+            steps {
+                // Checkout the Git repository
+                checkout([$class: 'GitSCM', branches: [[name: 'main']], userRemoteConfigs: [[url: '[https://github.com/Your github project link.git']]](https://github.com/Your github project link.git)])
+            }
+        }
+        stage ('Install requirements.txt if Python PIP') {
+            steps {
+                script {
+                    if (fileExists('requirements.txt')) {
+                        sh 'pip install -r requirements.txt'
+                    }
+                }
+            }
+        }
+        stage('RHDA Step') {
+            steps {
+                echo 'RHDA'
+                rhdaAnalysis consentTelemetry: true, file: 'manifestName.extension'
+            }
+        }
+    }
+}
+
+
+```
+
+#### Return Code From Plugin
 - It returns 3 different exit status code
-    - 0: Analysis is successful and there were no vulnerabilities found in the dependency stack.
-    - 1: Analysis encountered an error.
-    - 2: Analysis is successful and it found 1 or more vulnerabilities in the dependency stack.
+    - 0: SUCCESS - Analysis is successful and there were no vulnerabilities found with Severity that exceeded the highest severity Allowed  in the dependency stack.
+    - 1: ERROR -   Analysis encountered an error.
+    - 2: VULNERABLE - Analysis is successful, but it found 1 or more vulnerabilities that Their Severity Exceeds the Highest Severity Allowed in the dependency stack.
```

Comment:
Minor fix:

`...  that Their Severity Exceeds the Highest Severity Allowed ...` -->
`...  that their severity exceeds the highest severity allowed ...`